product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx

```diff
@@ -39,21 +53,29 @@ The restore operation is available for any cluster that has at least one availab
 #### Restore a single-node or primary/standby high-availability cluster
 
 1.  Select the cluster you want to restore on the [Clusters](https://portal.biganimal.com/clusters) page in the [BigAnimal portal](https://portal.biganimal.com). 
-2.  From **Quick Actions**, select **Restore**.
-3.  On the Restore Cluster page: 
+1.  From **Quick Actions**, select **Restore**.
+1.  On the Restore Cluster page: 
     1.  Fill in the required fields.
-    2.  To restore to the last possible recovery point, in the **Source** section, in the **Point in Time Restore** field, select **Now** on the calendar. Or, to restore further back in time, choose a timestamp. 
-    3.  Review your selections in **Cluster Summary** and select **Restore Cluster** to begin the restore process.  
-4.  The new cluster is now available on the [Clusters](https://portal.biganimal.com/clusters) page.
+    1.  To restore to the last possible recovery point, in the **Source** section, in the **Point in Time Restore** field, select **Now** on the calendar. Or, to restore further back in time, choose a timestamp. 
+    1.  Review your selections in **Cluster Summary** and select **Restore Cluster** to begin the restore process. 
+        - If you have opted for volume snapshots and 
+          - you are restoring the cluster in the same region as your source cluster, then BigAnimal chooses volume snapshots for restore over base backups. 
+          - you are restoring the cluster in the different region from your source cluster, then BigAnimal chooses the base backup for restore over volume snapshots.
+1.  The new cluster is now available on the [Clusters](https://portal.biganimal.com/clusters) page.
 
 #### Restore a distributed high-availability cluster
 
-1.  On the [Clusters](https://portal.biganimal.com/clusters) page in the [BigAnimal portal](https://portal.biganimal.com), select the cluster you want to restore.
-1.  Select **Quick Actions > Restore**.
+1. On the [Clusters](https://portal.biganimal.com/clusters) page in the [BigAnimal portal](https://portal.biganimal.com), select the cluster you want to restore.
+1. Select **Quick Actions > Restore**.
 1. In the **Cluster Settings** tab, enter a cluster name and password for your restored cluster. 
 1. Select **Next: Data Group**. 
 1. Select the **Node Settings** tab. 
 1. In the **Source** section, select **Fully Restore** or **Point in Time Restore**. A point-in-time restore restores the data group as it was at the specified date and time. 
 1. In the **Nodes** section, select **Two Data Nodes** or **Three Data Nodes**. For more information on node architecture, see [Distributed high availability](/biganimal/latest/overview/02_high_availability/distributed_highavailability/).
 1. Follow Steps 3-5 in [Creating a distributed high-availability cluster](../getting_started/creating_a_cluster/creating_a_dha_cluster/). 
```

Comment:
Seems like this page was a single flow before, and now it is split into sections. So maybe instead of painting to steps 3-5 we can point to a heading?